packages/kbn-apm-synthtrace/src/scenarios/logs_scenario_base.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { ScenarioBootstrap } from '../cli/scenario';
+import { RunOptions } from '../cli/utils/parse_run_cli_flags';
+import { IndexTemplateName } from '../lib/logs/templates';
+import { parseLogsScenarioOpts } from './helpers/logs_scenario_opts_parser';
+
+const logsScenarioBase: (runOptions: RunOptions) => { bootstrap: ScenarioBootstrap } = (
```

Comment:
I can see what you are trying to do here, avoiding duplicating code for all logs scenarios, but i am not sure this is right way forward. The way we envision the scenario files is that they are in themselves 1 complete scenario which means they have all the logic to bootstrap and generate. Doing this would break that vision. 

It is better to duplicate this code in all files. 

I don't have a realistic example at the moment, but imagine you wanting to do some additional stuff in bootstrap stage of only a few Logs scenarios. By splitting them like this, it will make it too much of a task to do that. Hence lets keep it simple in the same file